deepprofiler/learning/models.py

```diff
@@ -18,7 +18,11 @@ def make_regularizer(transforms, reg_lambda):
     return loss
 
 
-def create_keras_resnet(input_shape, targets, learning_rate=0.001, embed_dims=256, reg_lambda=10, is_training=True):
+def create_keras_resnet(input_shape, targets, config, learning_rate=0.001, embed_dims=256, reg_lambda=10, is_training=True):
```

Comment:
No need to pass the full `config` object here. Create a new parameter called `top_k` with a default value of 5, and let the user overwrite it.

Applies to all functions in this file.